book/pend.html

```diff
@@ -690,7 +690,7 @@ <h1>The Simple Pendulum</h1>
 
       Consider the dynamical system given by \begin{align*} \dot x_1 &= x_1
       (1-|\bx|) + x_2 \frac{x_1-|\bx|}{2|\bx|}, \\ \dot x_2 &= x_2 (1-|\bx|) -
-      x_1 \frac{x_1-|\bx|}{2|\bx|}, \end{align*} where $|\bx| =
+      x_1 \frac{x_1-|\bx|}{2|\bx|}, \end{align*} for $\bx\neq [0,0]^\top$, and $\dot \bx = [0,0]^\top$ for $\bx = [0,0]^\top$, where $|\bx| =
```

Comment:
i think this is confusing as written.  i had to read it a few times.  can we use \begin{cases} environment instead?